HdmiCecSink/HdmiCecSink.cpp

```diff
@@ -2470,7 +2477,12 @@ namespace WPEFramework
 
 					case CECDeviceParams::REQUEST_OSD_NAME :	
 					{
-						_instance->deviceList[logicalAddress].update(OSDName("NA"));
+						if ( _instance->deviceList[logicalAddress].m_isRequestRetry++ >= HDMICECSINK_REQUEST_MAX_RETRY )
```

Comment:
So this code chanegs is adding retry to OSD name. Could we add this to VENDOR ID as well?

Reply:
Vendor id is not that mandatory like physical address or osd name.so we really need not

Reply:
I know but providing vendor id reliably application write some logic for gaming console or something like that. So adding try there will help.

Reply:
To be failr I dont want to add it now and make the cec bus busy.As we don't know which all devices provide that information and which ones dont.When it really needed we can add.